robot/robot.py

```diff
@@ -38,23 +38,26 @@ def __init__(self, robotd_path=ROBOTD_ADDRESS):
             self.known_gamestates,
         ]
 
+        self._display_boards()
```

Comment:
I'm unfortunately fairly sure that the boards lists will be empty at this point. This is because we currently only initialise them lazily, something I'd not realised until I ran the full stack locally this week.

We could force the initialisation of the devices here, though that then leads to the wondering of whether the boards themselves should be logging their initialisation, rather than it being a global thing. If we had proper logging (waaay out of scope for this PR), I'd probably suggest doing both, under different loggers, with the logger here outputting to the main log and keeping the per-board messages for debugging.

Reply:
Agreed totally. I think this line (and therefore the `_display_boards` function) can be removed without issue - we still log when boards are connected to for the first time ("New board found: /var/robotd/power/xxx").